addons/mail/models/mail_channel.py

```diff
@@ -1093,7 +1093,7 @@ def create_group(self, partners_to, default_display_mode=False):
             'channel_last_seen_partner_ids': [Command.create({'partner_id': partner_id}) for partner_id in partners_to],
             'channel_type': 'group',
             'default_display_mode': default_display_mode,
-            'name': '',  # default name is computed client side from the list of members
+            'name': name,  # default name is computed client side from the list of members
```

Comment:
The comment on this line applied to the empty string default value that was there before, not the assignation. It should be moved as param doc instead (which is by the way currently missing).